### DISCUSSION TITLE
Imports vs Suggests

### DISCUSSION THREAD

**Discussion by jmgirard** [easystats/easystats#436] [General]
While teaching my intro to stats graduate course, it is commonly a source of confusion and frustration for students when easystats asks them to install an optional package. An example that came up multiple times this week was performance asking them to install patchwork. I understand the desire to keep the dependency footprint light, but the choice to make some of these important packages suggests instead of imports seems to be running contrary to the goal of making easystats "easy." As always, I love this package - just wanted to share some feedback from someone "in the trenches" using this stuff for teaching.

**Comment by bwiernik** [comment]
A usesful step for your teaching could be to instruct your students to run `easystats::install_suggested()`

**Reply by strengejacke** [reply]
I think the best is simply adding it to the _see_ imports, where we actually don't care much about dependencies (we import ggplot anyway, and _see_ is probably no package that other developers strictly rely on)

@IndrajeetPatil WDYT?

**Reply by bwiernik** [reply]
Agree with that one

**Reply by IndrajeetPatil** [reply]
Makes sense. Moved `{patchwork}` to Imports.
